README.rst

```diff
@@ -67,6 +67,18 @@ equal to the default threshold. To double the limit, use::
 NB: Z-projection is not supported for tiled images in OMERO
 (Images larger than 2000 * 2000 pixels per plane are tiled in iviewer).
 
+Known issues
+============
+
+For images with many channels (greater than approximately 30 channels), saving
+of rendering settings fails due to the length of the request string. See
+`#321 <https://github.com/ome/omero-iviewer/issues/321>`_. A work-around is to
+configure nginx to allow longer request strings. For example, to double the
```

Comment:
```suggestion
configure gunicorn to allow longer request strings. For example, to double the
```

I think the Nginx limit is configured separately

Reply:
Thanks @manics